Lib/test/test_socketserver.py

```diff
@@ -55,7 +55,7 @@ class ForkingUnixDatagramServer(socketserver.ForkingMixIn,
                                     socketserver.UnixDatagramServer):
         pass
 
-
+@test.support.requires_fork() # TODO: RUSTPYTHON, os.fork is currently only supported on Unix-based systems
```

Comment:
This is actually looking python test suite bug.
If you send this change without comment to CPython, they probably will accept it.

Reply:
Raising PR for this to cpython

Reply:
👍  https://github.com/python/cpython/pull/103717